src/test/java/tests/RepositorySelectorPopulationTests.java

```diff
@@ -47,16 +49,17 @@ public void noJsonFiles() {
     public void oneInvalidJsonFile() throws IOException {
         File invalidJson = new File("store/test/dummy-dummy.json");
         assert invalidJson.createNewFile();
-        RepoIO testIO = new RepoIO(true, true);
+        RepoIO testIO = TestController.createTestingRepoIO(Optional.of(new JSONStoreStub()));
```

Comment:
I think this tests suite is weird in the first place. It's called `RepositorySelectorPopulationTests` but has nothing to do with that? It's more like `StoreTests`, and in that case, all the tests should use the real `JSONStore`.